src/Reply.cpp

```diff
@@ -64,6 +64,22 @@ std::string Reply::ERR_NOSUCHNICK(const std::string &nick) const {
 	return nick + " :No such nick";
 }
 
+std::string Reply::RPL_WELCOME(const std::string &nickname,
+							   const std::string &username) const {
+	std::string result;
+	char dateStr[100];
+	std::time_t t = std::time(nullptr);
```

Comment:
```suggestion
	std::time_t t = std::time(nullptr);
```
nullptrはc++11なので使わないでください

Reply:
コンフリクトが発生するのでfeat irssiをマージしてから直したいと思います。

Reply:
忘れないようにissue立てておいて下さい